README.md

```diff
@@ -159,7 +159,62 @@ For details on all available API methods, see the [reference page](api_reference
 
 ## Examples
 
-As well as the following examples, you can take a look at our [examples directory](https://github.com/iotaledger/iota.js/tree/next/examples) for more.
+We have a list of test cases in the [`examples` directory](https://github.com/iotaledger/iota.js/tree/next/examples) that you can use as a reference when developing apps with IOTA.
+
+Here's how you could send a zero-value transaction, using the library. For the guide, see the [documentation portal](https://docs.iota.org/docs/client-libraries/0.1/how-to-guides/js/send-your-first-bundle).
+
+```js
+const Iota = require('@iota/core');
+const Converter = require('@iota/converter');
+
+// Connect to a node
+const iota = Iota.composeAPI({
+  provider: 'https://nodes.devnet.thetangle.org:443'
+});
+
+const depth = 3;
+const minimumWeightMagnitude = 9;
+
+// Define a seed and an address.
+// These do not need to belong to anyone or have IOTA tokens.
+// They must only contain a mamximum of 81 trytes
+// or 90 trytes with a valid checksum
+const address =
+  'HEQLOWORLDHELLOWORLDHELLOWORLDHELLOWORLDHELLOWORLDHELLOWORLDHELLOWORLDHELLOWOR99D';
+const seed =
+  'PUEOTSEITFEVEWCWBTSIZM9NKRGJEIMXTULBACGFRQK9IMGICLBKW9TTEVSDQMGWKBXPVCBMMCXWMNPDX';
+
+// Define a message to send.
+// This message must include only ASCII characters.
+const message = JSON.stringify({"message": "Hello world"});
+
+// Convert the message to trytes
+const messageInTrytes = Converter.asciiToTrytes(message);
+
+// Define a zero-value transaction object
+// that sends the message to the address
+const transfers = [
+  {
+    value: 0,
+    address: address,
+    message: messageInTrytes
+  }
+];
+
+// Create a bundle from the `transfers` array
+// and send the transaction to the node
+iota
+  .prepareTransfers(seed, transfers)
+  .then(trytes => {
+    return iota.sendTrytes(trytes, depth, minimumWeightMagnitude);
+  })
+  .then(bundle => {
```

Comment:
Could we add fetching of the message as well? Only the tail hash can be used, otherwise it's possible to obtain attachments of bundle with different message fragments.